lib/worker/src/rpc.rs

```diff
@@ -249,7 +274,7 @@ impl publications::callback::Server for UploadCallback {
 }
 
 /// Listens for RPC messages from Basin clients
-pub async fn listen<E: EVMClient>(
+pub async fn listen<E: EVMClient + 'static>(
```

Comment:
why do we have to change this lifetime annotation?

Reply:
not really sure. looks like the rustc from CI was on a newer version and complained about this